packages/markdown/remark/src/rehype-islands.ts

```diff
@@ -1,4 +1,6 @@
-import { SKIP, visit } from 'unist-util-visit';
+import { SKIP, visit as _visit } from 'unist-util-visit';
+
+const visit = _visit as (node: any, type: string, callback?: (node: any, index: number, parent: any) => any) => any;
```

Comment:
Oh my, is this due to the upgrade? Can you explain the reason? `unist-util-visit` is incompatible with the new version perhaps?

Would definitely want a comment above this explaining why this is needed.

Reply:
Good idea. How’s this?

https://github.com/withastro/astro/pull/2087/commits/584848b82e003b0cbe2d5d81556857c8244cb66f